src/graphql/account.rs

```diff
@@ -59,12 +61,24 @@ impl AccountQuery {
         first: Option<i32>,
         last: Option<i32>,
     ) -> Result<Connection<String, Account, AccountTotalCount, EmptyFields>> {
+        let (after, before, first, last) =
+            validate_and_process_pagination_params(after, before, first, last)?;
+
         query(
             after,
             before,
             first,
             last,
-            |after, before, first, last| async move { load(ctx, after, before, first, last).await },
+            |after, before, first, last| async move {
+                load(
+                    ctx,
+                    after,
+                    before,
+                    first.and_then(|f| f.to_i32()),
+                    last.and_then(|f| f.to_i32()),
```

Comment:
Let's avoid this casting. Let's stick to the previous code.

```rust
|after, before, first, last| async move { load(ctx, after, before, first, last).await },
```

The `query` function, which is a function provided by `async_graphql` crate, converts `first` and `last` to `usize`, so let's not re-convert it to `i32`. 

So, when `load` is called within the `query`, let's have `first` and `last` accept `Option<usize>`.

Reply:
Should the type casting occur here when I call `load_edges`? 

<img width="1512" alt="Screenshot 2024-04-04 at 8 45 44 PM" src="https://github.com/aicers/review-web/assets/102562226/8a49f64a-5fff-43f9-a81f-460af55c7b85">

Reply:
Sorry for not being so clear. That `graphql::load_edges` should also accept `Option<usize>` subsequently.

Reply:
Ahhhh ok! I think I get it!

Reply:
Done